content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,94 +1,121 @@
-=========================
-Create Bills of Materials
-=========================
+================================
+Create a Bill of Materials (BoM)
+================================
 
-A *Bill of Materials* is a document defining the quantity of each
-component required to manufacture a finished product. It also includes
-the routing and individual steps of the manufacturing process.
+A *Bill of Materials* (or *BoM*) is a document that defines the quantity of each component required
+to manufacture or deliver a finished product. It can also include various operations types and
+steps, and instructions for the individual step guidelines needed to complete a production process.
```

Comment:
A *Bill of Materials* (or *BoM*) is a document that defines the quantity of each component required
to manufacture (or deliver) a finished product. It can also include various operations types, 
steps, and instructions for the individual guidelines needed to complete a production process.